.travis.yml

```diff
@@ -1,13 +1,11 @@
 sudo: false
```

Comment:
Travis has a warning that this has no effect anymore. I guess you can remove it and update the PR to kick travis a little.